README.md

```diff
@@ -167,10 +167,10 @@ look for the webpack server at that host instead of the host where Docker is run
 You'll need to install the [yarn](https://yarnpkg.com/en/docs/cli/)
 package manager. You can do:
 
-    npm install -g yarn@0.19.1
+    node ./scripts/install_yarn.js
```

Comment:
I think most users will need to be root for this to work (`npm install -g`) but I am not user if nodejs would be installed for the user on MacOS.